backend/src/main/java/com/bakdata/conquery/models/datasets/Column.java

```diff
@@ -76,7 +76,10 @@ public Dataset getDataset() {
 	 */
 	@Override
 	public TrieSearch<FrontendValue> createTrieSearch(IndexConfig config, NamespaceStorage storage) {
-
 		return config.createTrieSearch(isGenerateSuffixes());
 	}
+
+	public void init() {
+		position = ArrayUtils.indexOf(getTable().getColumns(), this);
+	}
```

Comment:
Ich denke, dass es hier sauberer ist, wenn wir den Setter `setTable()` überschreiben. Den sollte Jackson dann auch nutzen können.

Reply:
oha nice! Gute idee!

Reply:
hab es gerade ausprobiert, das geht nicht, weil columns erst initialisiert wird nach der column construction :<

Reply:
War die lazy init denn der Grund für die vielen Lambdas im histo, oder wie kommt es zu dem PR?

Reply:
nein, aber wir haben beide schon unabhängig in flamegraphs von calculateCblocks sehr viel getColumn calls gesehen, die ja eigentlich komplett überflüssig sind wenn die methode ordentlich inlined wird (was sie anscheinend nicht wird).

Reply:
oki